app/views/multifactor_auths/mfa_prompt.html.erb

```diff
@@ -17,15 +17,30 @@
       </div>
   <% end %>
 
-  <% if @user.totp_enabled? %>
-    <%= form_tag @form_mfa_url, method: :post do %>
-      <div class="text_field">
-        <%= label_tag :otp, t("multifactor_auths.otp_code"), class: "form__label" %>
-        <%= text_field_tag :otp, "", class: "form__input", autofocus: true, autocomplete: :off %>
-      </div>
-      <div class="form_bottom">
-        <%= submit_tag t("authenticate"), data: {disable_with: t("form_disable_with")}, class: "form__submit" %>
+  <% if @user.totp_enabled? || @user.webauthn_only_with_recovery? %>
+    <div class="mfa__option">
+      <% if @user.totp_enabled? %>
+        <h2 class="page__subheading--block"> <%= t("multifactor_auths.otp_code") %></h2>
+      <% elsif @user.webauthn_only_with_recovery? %>
+        <h2 class="page__subheading--block"> <%= t("multifactor_auths.recovery_code") %></h2>
+      <% end %>
+      <div class="t-body">
+        <p><%= t("multifactor_auths.recovery_code_html") %></p>
       </div>
-    <% end %>
+      <%= form_tag @form_mfa_url, method: :post do %>
+        <div class="text_field">
+          <% if @user.totp_enabled? %>
+            <%= label_tag :otp, t('multifactor_auths.otp_or_recovery'), class: 'form__label' %>
+            <%= text_field_tag :otp, '', class: 'form__input', autofocus: true, autocomplete: :off %>
+          <% elsif @user.webauthn_only_with_recovery? %>
+            <%= label_tag :otp, t('multifactor_auths.recovery_code'), class: 'form__label' %>
```

Comment:
nit: do we need the label tag here? It looks a bit redundant that the heading is also `Recovery code`
<img width="379" alt="Screenshot 2023-06-14 at 10 45 22 AM" src="https://github.com/rubygems/rubygems.org/assets/42748004/4ae0f222-85ab-4e23-b700-639d190c0dcc">

Reply:
I believe for accessibility it's better to keep the label tag there.

Reply:
I think the label will still be associated with the text field even if hidden?

Reply:
Herm ... I agree with Jenny. It does feel redundant 😅 . I think Eric & I had noted this while prototyping too. Staring at this longer, I'm inclined to advise dropping the label as well. 

From an a11y standpoint, we can remove the label and add the `aria-label` attribute on the `input` (e.g. `aria-label="Recovery code"`).

Reply:
Added aria label! https://github.com/rubygems/rubygems.org/pull/3859/commits/4123c0973a120729ab95c7281def07512c41d624